modules/managers.py

```diff
@@ -0,0 +1,35 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect('../cinema.sqlite')

```

Comment:
The file 'cinema.sqlite' is being used here, and it seems like it is a database file. According to the checklist, you should not push db files. Consider adding this file to .gitignore to avoid accidentally committing it to the repository.